tests/flow/test_index_scans.py

```diff
@@ -276,3 +276,10 @@ def test11_single_index_multiple_scans(self):
         query_result = redis_graph.query(query)
         # Two new nodes should be created.
         self.env.assertEquals(query_result.nodes_created, 2)
+
+    def test12_remove_scans_before_index(self):
```

Comment:
@jeffreylovitz I would have add a comment on this test pointing to the github issue